plugins/collaajUpdatePlugin/lib/api/KalturaCaptureSpaceUpdateResponse.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+class KalturaCaptureSpaceUpdateResponse extends KalturaObject {

```

Comment:
What's the purpose of this class, it's just wrapping info.
Why won't it contain the content of info instead?